source/github-helpers/index.ts

```diff
@@ -22,14 +22,25 @@ Example tag content on public repositories: https://github.com/sindresorhus/refi
 Example tag content on private repositories https://github.com/private/private/commits/master.atom?token=AEAXKWNRHXA2XJ2ZWCMGUUN44LM62
 */
 export const getCurrentBranch = (): string | undefined => {
+	// Example title: 'tejanium/refined-github:bra/nch' or just 'local-branch`
+	if (pageDetect.isPRFiles()) {
```

Comment:
This weird. The comment above specifically mentions "Subtly incompatible with isPR", but we're still using it there. Maybe `getCurrentBranch` should just _not_ be used there since PRs have 2 branches anyway, this function is too generic.

Anyway, I'm working on this

Reply:
> "Subtly incompatible with isPR"

What that means is that on PR's the current branch can also be a commit.

> Anyway, I'm working on this

👍